client/components/GuestPicker.jsx

```diff
@@ -0,0 +1,103 @@
+import React from 'react'
+import styled from 'styled-components';
+
+const GuestContainer = styled.div`
+  width: 280px;
+  height: 320px;
+  border: 1px solid rgb(221, 221, 221);
+  border-radius: 4px;
+  box-shadow: rgba(0, 0, 0, 0.15) 0px 2px 6px;
+  position: absolute;
+  transform: translate(20px, 0px);
+  z-index: 999;
+  background: #FFFFFF;
+  padding: 16px;
+  display: inline-block;
+`
+
+const GuestSelectLeft = styled.div`
+  width: 20px;
+  height: 20px;
+  border: 1px solid #717171;
+  border-radius: 32px;
+  text-align: center;
+  margin-top: 0px;
+  float: left;
+  cursor: pointer;
+`
+
+const GuestSelectRight = styled.div`
+  width: 20px;
+  height: 20px;
+  border: 1px solid #717171;
+  border-radius: 32px;
+  text-align: center;
+  margin-top: 0px;
+  float: right;
+  cursor: pointer;
+`
```

Comment:
Consider extending styled components. All of these properties are the same except for one. https://stackoverflow.com/questions/52966623/how-to-extend-styled-components --> Second answer explains it well. I started refactoring a little and its dried up my code a lot!

Reply:
i'd love to get rid of some of these components.  thank you!  i'll take a look.